public/components/security/roles-mapping/components/rule-editor.tsx

```diff
@@ -111,17 +111,30 @@ export const RuleEditor = ({ save, initialRule, isLoading, isReserved, internalU
     rulesTmp.splice(id, 1);
     setRules(rulesTmp);
   };
+  
+  const getRulesFromJson = (jsonRule) => {
+    if (jsonRule !== '{}') {
```

Comment:
I tested this PR and the behaviour that @Desvelao has commented would be solved changing `if (jsonRule !== '{}')`  to ` if (jsonRule !== '') `